os/include/stdlib.h

```diff
@@ -551,10 +551,13 @@ lldiv_t lldiv(long long numer, long long denom);
  */
 
 /**
- * @cond
- * @internal
+ * @ingroup STDLIB_LIBC
+ * @brief compute the quotient and remainder of an integer division
+ * @details @b #include <stdlib.h> \n
+ * POSIX API (refer to : http://pubs.opengroup.org/onlinepubs/9699919799/)
```

Comment:
I can not find this in POSIX site. Is this right?

Reply:
yes, it has been removed since `POSIX.1-2008`. I will make this as an internal API